docs/source/data_acquisition/DataProv-MSSentinel.rst

```diff
@@ -59,186 +81,230 @@ The MS Sentinel connection settings are stored in the
 
 If you only use a single workspace, you only need to create a ``Default`` entry and
 add the values for your *WorkspaceID* and *TenantID*. You can add other entries here,
-for example, SubscriptionID, ResourceGroup. These are recommended but not required
-for the QueryProvider (they may be used by other *MSTICPy* components however).
-
-.. note:: The property names are spelled differently to the values in the
-   *config.json* so be sure to enter these as shown in the example. These
-   names are case-sensitive.
+for example, SubscriptionID, ResourceGroup. These are not required for the data
+queries but are recommended since they are used by other *MSTICPy* components.
 
-.. note:: The section names (Default, Workspace1 and TestWorkspace) do
-   not have to be the same as the workspace name - you can choose friendlier
-   aliases, if you wish.
+If you use multiple workspaces, you can add further entries here. The key for
+each entry (e.g. ``Workspace1`` or ``TestWorkspace`` in the example above)
+is normally the name of the Azure Sentinel workspace but
+you can use any name you prefer. You use this entry name when connecting
+to a workspace.
 
-If you use multiple workspaces, you can add further entries here. Each
-workspace entry is normally the name of the Azure Sentinel workspace but
-you can use any name you prefer.
 
-Sentinel Configuration in *config.json*
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Loading a QueryProvider for Microsoft Sentinel
+----------------------------------------------
 
-When you load a notebook from the MS Sentinel UI a configuration file *config.json*
-is provisioned for you with the details of the source workspace populated in
-the file. An example is shown here.
+.. code:: ipython3
 
-.. code:: json
+    qry_prov = QueryProvider(
+        data_environment="MSSentinel_New",
+    )
 
-    {
-        "tenant_id": "335b56ab-67a2-4118-ac14-6eb454f350af",
-        "subscription_id": "b8f250f8-1ba5-4b2c-8e74-f7ea4a1df8a6",
-        "resource_group": "ExampleWorkspaceRG",
-        "workspace_id": "271f17d3-5457-4237-9131-ae98a6f55c37",
-        "workspace_name": "ExampleWorkspace"
-    }
+    # or just
+    qry_prov = QueryProvider("MSSentinel_New")
 
-If no *msticpyconfig.yaml* is found *MSTICPy* will automatically look for a
-*config.json* file in the current
-directory. If not found here, it will search the parent directory and in all
-its subdirectories. It will use the first *config.json* file found.
+Optional parameters for the Sentinel QueryProvider
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+``timeout`` : int (seconds)
 
-Loading a QueryProvider for Microsoft Sentinel
-----------------------------------------------
+Specify a timeout for queries. Default is 300 seconds.
```

Comment:
Are there specific values worht mentioning here? For instance, is 0 the value to "disable" the timeout, or is there a maximum value?